hawkbit-repository/hawkbit-repository-test/src/main/java/org/eclipse/hawkbit/repository/test/util/AbstractIntegrationTest.java

```diff
@@ -292,7 +292,11 @@ public void before() throws Exception {
 
     @After
     public void cleanCurrentCollection() throws IOException {
-        FileUtils.deleteDirectory(new File(artifactFilesystemProperties.getPath()));
+        try {
+            FileUtils.deleteDirectory(new File(artifactFilesystemProperties.getPath()));
+        } catch (final IOException e) {
```

Comment:
`cleanCurrentCollection()` don't throw an `IOException` anymore. So please remove it from the signature.